src/api/built-in-components.md

```diff
@@ -27,6 +41,27 @@
   <component :is="href ? 'a' : 'span'"></component>
   ```
 
+  The built-in components `KeepAlive`, `Transition`, `TransitionGroup`, and `Teleport` can all be passed to `is` but you must register them if you want to pass them by name. e.g.: 
```

Comment:
**nitpick**: Add comma

**suggestion:** Remove abbreviation to improve comprehension for those unfamiliar with i.e., e.g., etc.

```suggestion
  The built-in components `KeepAlive`, `Transition`, `TransitionGroup`, and `Teleport` can all be passed to `is`, but you must register them if you want to pass them by name. For example:
```